### ISSUE TITLE
Info for new players:

### ISSUE THREAD

**Issue by qwop9992** [4onenm/AwSW-Bangok#3]
So I've just started the game and heard there weren't any real lewd scenes, so I poked around for mods and found this one. I want to install it to my game (I'm on like, chapter 3) but I want to make sure of two things:

1) do these scenes interfere with the story/progression at all? Or can I get the whole experience of the vanilla game, just +lewds.

2) do the scenes, once enabled in settings, appear naturally? Like, can I just stumble upon them if I make the right choices? 

I know most of this is probably mentioned in the guides or parts of this post, but I'm trying not to spoiler too much of the game for myself. I'd super appreciate the answer, and I'm sure other players in similar situations would be too!

**Comment by 4onenm** [comment]
1. Released scenes are designed to fit somewhat naturally into the flow of the story, so they do not block progression. (Note that this does not include "In-Development Scenes" which you can enable from the mod settings menu, as those can break individual playthrough saves (luckily not the persistent saves, though.))
2. Yep! They're pretty natural, though they don't appear everywhere they would make sense (e.g. there are base-game, un-modded mentions of lewdness at specific points in one or two character routes that _do not_ have associated BangOk content... yet. Feel free to contribute, once you get there and if you so desire!)

Tl;dr:
1. The scenes do not interfere
2. Yes.
